interpretation/ww_head.py

```diff
@@ -34,6 +34,10 @@ def process(message, isGameMaster = False):
         msg += "\nThe bot understands both mentions and emojis linked to players."
         return [Mailbox().respond(msg,True)]
 
+    '''testpoll'''
+    if is_command(message,['poll','testpoll']):
+        return [Mailbox().new_poll(message.channel.id,'kill',message.author.id,message.content.split(' ',1)[1])]
+
```

Comment:
This function is obviously there to test. It has no help explanation, nor does it know what to do if there is no extra argument given after the command.